docker-compose.yml

```diff
@@ -14,7 +14,7 @@ backend:
     DRAGONFLY_SECRET: c2jq0a9cj039jc01mamc09qjc930ur10fjafj0a9fj09fj23qnweocn0qwf0qfjq
     MANSHAR_DB_1_PORT_5432_TCP_ADDR: db
   volumes:
-    - backend/:/manshar-backend
+    - ./backend/:/manshar-backend
```

Comment:
Reference:
`Volumes that don't start with a path indicator (. or /) will now be treated as a named volume. Previously this was a warning.`
https://github.com/docker/compose/releases/tag/1.5.0rc1